pkgs/top-level/all-packages.nix

```diff
@@ -10253,6 +10253,8 @@ with pkgs;
 
   kb = callPackage ../tools/misc/kb { };
 
+  dooit = callPackage ../tools/misc/dooit { };
```

Comment:
This file should be alphabetically ordered.

Reply:
sure.